keras_cv/layers/preprocessing/random_rotation.py

```diff
@@ -78,6 +78,8 @@ class RandomRotation(BaseImageAugmentationLayer):
       bounding_box_format: The format of bounding boxes of input dataset. Refer
         https://github.com/keras-team/keras-cv/blob/master/keras_cv/bounding_box/converters.py
         for more details on supported bounding box formats.
+      segmentation_classes: The number of classes in the input segmentation mask.
+        Required iff augmenting data with sparse (non one-hot) segmentation masks.
```

Comment:
have we determined that we will only be supporting masks in the form of one-hot images?  (i.e. no polygon formats)

I think this is a pretty reasonable choice, mainly just curious.

Reply:
Yes -- given that this is how predictions are made in segmentation, I think they should live in the training pipeline like this. For datasets with polygonal masks, we can offer preprocessing code to transform those into pixel masks.